tests/scripts/helm.sh

```diff
@@ -1,26 +1,33 @@
 #!/bin/bash +e
+
 scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+temp="/tmp/rook-tests-scripts-helm"
+
+HELM="helm"
 
 install() {
-    #Download and unpack helm
-    local dist
-    dist="$(uname -s)"
-    # shellcheck disable=SC2021
-    dist=$(echo "${dist}" | tr "[A-Z]" "[a-z]")
-    wget "https://storage.googleapis.com/kubernetes-helm/helm-v2.6.0-${dist}-amd64.tar.gz"
-    tar -zxvf "helm-v2.6.0-${dist}-amd64.tar.gz"
-    sudo mv "${dist}-amd64/helm" /usr/local/bin/helm
+    if ! helm_loc="$(type -p "helm")" || [[ -z ${helm_loc} ]]; then
+        # Download and unpack helm
+        local dist
+        dist="$(uname -s)"
+        # shellcheck disable=SC2021
+        dist=$(echo "${dist}" | tr "[A-Z]" "[a-z]")
+        mkdir -p "${temp}"
+        wget "https://storage.googleapis.com/kubernetes-helm/helm-v2.6.0-${dist}-amd64.tar.gz" -O "${temp}/helm-v2.6.0-${dist}-amd64.tar.gz"
+        tar -C "${temp}" -zxvf "${temp}/helm-v2.6.0-${dist}-amd64.tar.gz"
```

Comment:
I see `amd64` is hardcode here, can we make it support other arch like `arm64` since I find that that link has `helm-v2.6.0-linux-arm64.tar.gz`, is that correct?

Reply:
If you have a well working mechanism to get the arch from "any" Linux system (not Debian specific), sure.

A simple solution would be to introduce an `ARCH` variable which can be overridden from the outside.

Reply:
Can we detect the ARCH in runtime such as `uname -m` in the script, thus if I run this script on my arm64 server, it will download the `arm64` version automatically, am I missing something?

Reply:
My Linux 64-bit Laptop returns: `x86_64` (not `amd64`).
A Scaleway ARM64-2GB server returns: `aarch64`.

What does `uname -m` output on your machine?

I will probably add the `ARCH` variable to allow manual override and as the default use `uname -m` then with conditionals for the "current" architectures, e.g., `aarch64` to `arm64`, `x86_64` to `amd64`, some other arches (besides 32-bit `i386`/`i686` and so on; if there are builds  available for them).

Reply:
In my local arm64 machine, the `uname -m` outputs `aarch64`, so we need a method to translate the `aarch64` to `arm64`, the same as `x86_64` does.